SCons/Tool/MSCommon/__init__.py

```diff
@@ -30,20 +30,23 @@
 import SCons.Platform.win32
 import SCons.Util
 
-from SCons.Tool.MSCommon.sdk import mssdk_exists, \
-                                    mssdk_setup_env
+from SCons.Tool.MSCommon.sdk import mssdk_exists, mssdk_setup_env
 
-from SCons.Tool.MSCommon.vc import msvc_exists, \
-                                   msvc_setup_env, \
-                                   msvc_setup_env_once, \
-                                   msvc_version_to_maj_min, \
-                                   msvc_find_vswhere
+from SCons.Tool.MSCommon.vc import (
```

Comment:
> __[Flake8]__ 'SCons.Tool.MSCommon.vc.msvc_find_vswhere' imported but unused ([view](https://sider.review/gh/repos/104670160/pulls/4080#issue-6441662))
>
> <details>
>
> | Rule |
> | ---- |
> | `F401` |
>
> You can [close this issue](https://sider.review/gh/repos/104670160/pulls/4080#issue-6441662) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

Okay, so based on the pinciple that they're imported to the main file of MSCommon so that they can be imported by other modules without reaching further inside this module, I'm going to mark all of these closed.